Source/Scene/ModelExperimental/ModelMatrixUpdateStage.js

```diff
@@ -72,6 +75,18 @@ function updateRuntimeNode(runtimeNode, sceneGraph, transformToRoot) {
         drawCommand.modelMatrix,
         drawCommand.boundingVolume
       );
+
+      const cullFace = ModelExperimentalUtility.getCullFace(
```

Comment:
Also update the tests for this pipeline stage to ensure it's updating the render state correctly.